pkg/controller/direct/cloudbuild/workerpool_controller.go

```diff
@@ -135,10 +135,23 @@ func (m *model) AdapterForObject(ctx context.Context, reader client.Reader, u *u
 	if err != nil {
 		return nil, err
 	}
+
+	projectIDAndNumer := &refs.ProjectIDAndNumer{ID: projectRef.ProjectID}
+	network, found, err := unstructured.NestedString(u.Object, "status", "observedState", "networkConfig", "peeredNetwork")
+	if err == nil && found {
+		tokens := strings.Split(network, "/")
+		if len(tokens) == 5 && tokens[0] == "projects" {
+			projectIDAndNumer.Number = tokens[1]
+			if err != nil {
```

Comment:
I think this is left over from parsing the number. Hopefully we can do that again if we change the mock.